styles/style.css

```diff
@@ -0,0 +1,378 @@
+* {
+    font-size: 100%;
+    margin: 0;
+}
+
+header {
+    background-image: url("../images/header-image.jpg");
+    background-position: right;
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+
+}
+
+header h1 {
+    color: rgb(18, 146, 18);
+    font-size: 1.25rem;
+}
+
+header p {
+    color: white;
+}
+
+.header-left {
+    padding-left: 4rem;
+    padding-top: 1.25rem;
+
+}
+
+.header-right {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    padding-right: 4rem;
+    padding-top: 2%;
+    padding-bottom: 9rem;
+}
+
+.cart {
+    width: 20%;
+}
+
+.pay {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 1rem;
+    justify-content: flex-end;
+    align-content: center;
+}
+
+.pay p {
+    margin-top: 2rem;
+}
+.pay img {
+    margin-right: 1rem;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+    font-weight: bold;
+}
+
+nav li,
+footer li {
+    list-style-type: none;
+    padding: 1rem;
+
+}
+
+nav ul {
+    margin-left: 10rem;
+
+}
+
+nav ul,
+footer ul {
+    display: flex;
+    flex-direction: row;
+
+}
+
+h2 {
+    font-size: 1.5rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    padding-left: 15rem;
+}
+
+.adopt h2 {
+    padding-left: 0rem;
+}
+
+button {
+    text-align: center;
+    color: white;
+    background-color: rgb(18, 146, 18);
+    font-weight: bold;
+}
+
+.dogs button {
+    margin: 1rem 2rem;
+    padding-top: .5rem;
+    padding-bottom: .5rem;
+    width: 80%;
+    border: solid 1px rgb(18, 146, 18)
+
+}
+
+.center button  {
+    width: 10%;
+    margin-left: 50rem;
+
+}
+
+.more-dogs {
+    background-color: lightgrey;
+    color: navy;
+    font-weight: bold;
+    border: solid 1px lightgrey;
+    width: 30%;
+    margin-top: 6rem;
+    padding-top: .5rem;
+    padding-bottom: .5rem;
+    
+
+}
+
+.submit button {
+    color: navy;
+    font-weight: bold;
+    background-color: lightgrey;
+    width: 10%;
+    margin-left: 15rem;
+}
+
+.dogs img {
+    width: 100%;
+}
+
+.dogs article {
+    border: solid lightgrey;
+}
+
+.dogs h3,
+.dogs h4,
+.dogs p,
+.dogs button {
+    padding-right: .5rem;
+    padding-left: .5rem;
+    padding-bottom: .5rem;
+}
+
+.adopt {
+    display: grid;
+    grid-template-columns: 30% 40%;
+    grid-template-rows: 25% 75%;
+    grid-column-gap: 5%;
+    grid-row-gap: 5%;
+    margin-left: 15rem;
+    margin-right: 10rem;
+    margin-bottom: 5rem;
+    padding-top: 3rem;
+}
+
+.adopt img {
+    width: 100%;
+}
+
+.dogs {
+    display: grid;
+    grid-template-columns: 20% 20% 20% 20%;
+    grid-template-rows: auto auto;
+    grid-column-gap: 3%;
+    grid-row-gap: 5%;
+    margin-left: 15rem;
+    padding-bottom: 3rem;
+}
+
+.blog {
+    display: grid;
+    grid-template-columns: 30% 40%;
```

Comment:
I would advise using flexbox over CSS grid for most things. In the case of your dog grid, you can go either way. In this case, you'll find using flexbox will simplify your code and make your site much more responsive. Here is a cheat sheet I always use for referencing flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/